Library/Homebrew/keg.rb

```diff
@@ -208,6 +208,7 @@ def ==(other)
 
   def empty_installation?
     Pathname.glob("#{path}/**/*") do |file|
+      return false if file.executable?
       next if file.directory?
       basename = file.basename.to_s
       next if Metafiles.copy?(basename)
```

Comment:
Sorry, I could not understand "look only in the `prefix`". What should I look in the `prefix` instead of checking whether files are executable?